modules/virt-creating-storage-class.adoc

```diff
@@ -2,107 +2,17 @@
 //
 // * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
 
-:_content-type: PROCEDURE
+:_content-type: CONCEPT
 [id="virt-creating-storage-class_{context}"]
 = Creating a storage class
```

Comment:
IIRC, a concept title isn't supposed to start with a gerund. Maybe "About creating storage classes" or something like that? (also would need to update the ID and possibly the filename)

Reply:
Yes. I meant to fix that and forgot.